### ISSUE TITLE
Help compiling app in windows

### ISSUE THREAD

**Issue by mapsgeek** [crcn/tandem-old#694]
hi, i mainly work with angular and don't have much knowledge of react but trying to play with tandem and explore it, however i can't build " compile " the application, in tutorials i see that the mac version has a build button which maybe will come later into windows but i tried to follow installing the 'paperclip-react-compiler' and run commands `npm run build` after `npm install` for sure using the react default template from tandem and also tried with some of the tutorials in the repo.
first in vscode it doesn't recognize and can't find module 'view.pc' 
![image](https://user-images.githubusercontent.com/10188264/52206722-ce640900-2883-11e9-96a3-b81b59395c05.png)
when i run `npm run build`, it logs code which looks like this
```
import * as React from "react";
import ApplicationController0, {Props as ApplicationController0Props} from "./controller";

type TextProps = {
  text?: string;
} & React.HTMLAttributes<any>;

type ElementProps = {
  ref?: any;
} & React.HTMLAttributes<any>;

export type BaseApplicationProps = {
  appContentProps?: TextProps;
  textProps?: TextProps;
} & ElementProps;

export type _5ae928e83Props = ApplicationController0Props;
export const Application: (props: ApplicationController0Props) => React.ReactElement<ApplicationController0Props>;
````
sorry for the nooby question but i feel that we need a better documentation or tutorial on this, if you can help me figure it out so i can start trying to build something with tandem.

**Comment by crcn** [comment]
Try changing the `package.json` build script to:

```
paperclip-react-compiler \"src/**/*.pc\" --definition --write; webpack
```

I'll need to verify when I have more time later today, but `src/**/*.pc` should be wrapped around in `"` for Windows. 

I'll ship out a fix a bit later today.

**Comment by mapsgeek** [comment]
same output as above

**Comment by crcn** [comment]
This should work:

```
paperclip-react-compiler \"src/**/*.pc\" --definition --write && webpack
```

The `;` was also causing issues. Just published this fix under [v10.1.31](https://github.com/tandemcode/tandem/releases/tag/untagged-ba18d332838deed9c725)

>  i see that the mac version has a build button which maybe will come later into windows

Yeah, I the Windows version will have the build button soon, I just need to figure out where to put it. 

> sorry for the nooby question but i feel that we need a better documentation or tutorial on this,

Nah, this is my bad 🙈. The documentation is fresh, and the app is buggy. I'm working through this stuff as fast as I can though, thanks for working around the issues! 🙇‍♂️

Is there anything you were confused about so far when using the app or docs?

**Comment by mapsgeek** [comment]
Thank you, new release works as expected and compiles to `/lib`, i totally understand the state of tandem and i have been following since i knew about the project, regarding the docs it was just about the `compiling part` due to the bug which i didn't know exactly what to do with the logged output.
now i think it's up to getting familiar  with the app and some react knowledge, i know the road still long :)
